tools/dprint_config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lineWidth": "80",
+  "indentWidth": "2",
+  "nextControlFlowPosition": "sameLine",
+  "bracePosition": "sameLine"
+}
```

Comment:
You can replace this with...

```json
{
    "deno": true
}
```

...to have the same config (I renamed this from "prettier" to "deno" because it was different on trailing commas).

Then you may want to test out and provide feedback on the new `preferSingleLine` option:

```json
{
    "deno": true,
    "preferSingleLine": true
}
```

...I'm still working on improving that, but I'll turn it on soon in the deno config because it's what prettier does.

Main things for me still are:

1. More `preferSingleLine` config.
2. Improve binary expression formatting (ex. `1 + 2`)